tests/raw_query/models.py

```diff
@@ -32,6 +32,10 @@ class Coffee(models.Model):
     price = models.DecimalField(max_digits=10, decimal_places=2, default=0)
 
 
+class Tea(models.Model):
```

Comment:
I'm inclined to use a model name that's more explanatory about its purpose such as `MixedCaseIDColumn`.